proto/internal.proto

```diff
@@ -161,6 +161,19 @@ message InternalTruncateLogResponse {
   optional ResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
 
+message InternalChangeReplicasRequest {
+  optional RequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  optional int32 node_id = 2 [(gogoproto.nullable) = false,
+      (gogoproto.customname) = "NodeID", (gogoproto.customtype) = "NodeID"];
+  optional int32 store_id = 3 [(gogoproto.nullable) = false,
+      (gogoproto.customname) = "StoreID", (gogoproto.customtype) = "StoreID"];
+  optional bool remove = 4 [(gogoproto.nullable) = false];
```

Comment:
I was hoping we'd be able to encapsulate the observer status within raft, but yeah, I'll make it an enum for future-proofing. Probably better to make a new enum here than refer to the one in `raftpb`.